pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -127,6 +127,9 @@ message MessageMetadata {
     // transaction related message info
     optional uint64 txnid_least_bits = 22 [default = 0];
     optional uint64 txnid_most_bits = 23 [default = 0];
+
+    /// Add last sequence id to support batch message with external sequence id
+    optional uint64 last_sequence_id = 24 [default = 0];
```

Comment:
It seems to me that we use `highest` sequence id at the client side, but use `last` at the wire protocol and broker side. Can we make it consistent by just using `highest` across the places?